src/Impostor.Server/Net/State/Game.Data.cs

```diff
@@ -348,6 +348,11 @@ private async ValueTask OnSpawnAsync(ClientPlayer sender, InnerNetObject netObj)
 
                 case InnerPlayerInfo playerInfo:
                 {
+                    if (!IsHostAuthoritive && await sender.Client.ReportCheatAsync(new CheatContext(nameof(GameDataTag.SpawnFlag)), CheatCategory.ProtocolExtension, "Spawning NetworkedPlayerInfo as vanilla host"))
```

Comment:
Hmm, I'm not sure if ProtocolExtension is the best category to use here, I think Ownership fits better but it's also not a perfect fit.